app/assets/stylesheets/article-show.scss

```diff
@@ -31,6 +31,97 @@
   height: 10px;
 }
 
+.mod-actions-menu {
+  position: fixed;
+  top: var(--header-height);
+  right: 0;
+  height: calc(100vh - 56px);
+  z-index: 200;
```

Comment:
Does this really need to be set to 200? If not, consider adding a z-index utility class on the DOM element that has the `.mod-actions-menu` on it, i.e. `z-50` (z-index: 50) or lower if possible.

Othere z-index utilty classes available.

```
z-10
z-20
z-30
z-40
z-50
```

Reply:
Hmm sorry how I would do this? I tried `z-index: var(--z-10)` but that didn't seem to work. Also looked at `_generator.scss` but just got confused because my front end game is weak :(